src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -51,16 +50,6 @@
   private File datastorePath;
 
   @Override
-  public List<File> getAppYamls() {
-    return appYamls;
-  }
-
-  public void setAppYamls(List<File> appYamls) {
-    this.appYamls = appYamls;
-  }
-
-
-  @Override
   public List<File> getServices() {
```

Comment:
Might want to add some Javadoc here that mentions these should be directories for flexible and app.yaml's for standard (or vice-versa)?